mri/operators/fourier/test_fourier_adjoint.py

```diff
@@ -196,8 +196,14 @@ def test_similarity_stack_3D(self):
                 # and N!=Nz
                 Nz = 16
                 _mask = np.random.randint(2, size=(N, N))
-                _mask3D = np.asarray([_mask for i in np.arange(Nz)])
-                _samples = convert_mask_to_locations(_mask3D.swapaxes(0, 2))
+
+                # Generate random mask along z
+                sampling_z = np.random.randint(2, size=Nz)
+                _mask3D = np.zeros((N, N, Nz))
+                for idx, acq_z in enumerate(sampling_z):
+                    _mask3D[:, :, idx] = _mask * acq_z
+                _samples = convert_mask_to_locations(_mask3D)
```

Comment:
I can see that they were here before, but I don't understand why these variable names begin with `_`.